src/jaegertracing/samplers/Config.cpp

```diff
@@ -36,11 +36,15 @@ void Config::fromEnv()
     const auto param = utils::EnvVariable::getStringVariable(kJAEGER_SAMPLER_PARAM_ENV_PROP);
     if (!param.empty()) {
         std::istringstream iss(param);
-        int paramVal = 0;
+        double paramVal = 0;
         if (iss >> paramVal) {
             _param = paramVal;
         }
```

Comment:
Probably should log a warning if value cannot be parsed. But it's not in scope for this PR.